exporter/src/client/grpc_client.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(dead_code)]
+
```

Comment:
Note related to the pr - we should probably remove "#![allow(dead_code)]" from the module scope. We can add it where needed, if needed.